distributed/rpc/ddp_rpc/main.py

```diff
@@ -145,9 +150,16 @@ def run_worker(rank, world_size):
         for fut in futs:
             fut.wait()
     elif rank <= 1:
+        if torch.accelerator.is_available():
+            acc = torch.accelerator.current_accelerator()
+            device = torch.device(acc)
+        else:
+            device = torch.device("cpu")
+        backend = torch.distributed.get_default_backend_for_device(device)
+        torch.accelerator.device_index(rank)
```

Comment:
how setting `torch.accelerator.device_index(rank)` works for "cpu"? why it's not under `torch.accelerator.is_available()`?